assets/agenda/utils.js

```diff
@@ -405,6 +405,25 @@ export function getInternalNotesFromCoverages(item) {
     return internalNotes;
 }
 
+/**
+ * Get list of subjects
+ *
+ * @param {Object} item
+ * @return {Array}
+ */
+export function getSubjects(item) {
+    let subjects = get(item, 'event.subject') || [];
+    const planningItems = get(item, 'planning_items', []);
+    (planningItems || []).forEach(p => {
+        subjects = subjects.concat(p.subject || []);
+        get(p, 'coverages', []).forEach(c => {
+            subjects = subjects.concat(c.subject || []);
+        });
+    });
+
+    return uniqBy(subjects.filter(subject => !isEmpty(subject)), 'qcode');
```

Comment:
I think in newsroom we store subjects with `code` not `qcode`

Reply:
Actually, in wire we missed that coversion. Also by checking the `push` again I just realized that we don't need to check every planning item. So I'll change it again.